frontend/src/components/explorer/gemBrowser/Compartment.vue

```diff
@@ -130,16 +130,14 @@ export default {
     async setup() {
       this.showLoaderMessage = 'Loading compartment data';
```

Comment:
To me, it doesn't look like this is the right place for this.

```suggestion
```

Reply:
Where would be right for it?

Reply:
It was moved to the previous suggestion, but now irrelevant.